quickwit-directories/src/bundle_directory.rs

```diff
@@ -86,15 +86,15 @@ fn split_footer(file_slice: FileSlice) -> io::Result<(FileSlice, FileSlice)> {
 
 /// Return two slices for given split: `[body and bundle meta data] [hotcache]`
 pub fn get_hotcache_from_split(data: OwnedBytes) -> io::Result<OwnedBytes> {
-    let split_file = FileSlice::new(Box::new(data));
+    let split_file = FileSlice::new(Arc::new(data));
```

Comment:
nit: Next time, update tantivy in a separate PR to reduce the noise.

Reply:
ok